app/src/main/java/com/keylesspalace/tusky/network/AuthInterceptor.java

```diff
@@ -17,7 +34,6 @@
  */
 
 public final class AuthInterceptor implements Interceptor {
-
     AccountManager accountManager;
 
     public AuthInterceptor(AccountManager accountManager) {
```

Comment:
please give this class another name so it is clear it handles authentication and the switching of instances. I was very confused. Something like `InstanceSwitchAuthInterceptor` maybe?